config/environments/test.rb

```diff
@@ -33,6 +33,8 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.default_url_options = { host: "example.com" }
```

Comment:
What domain should we use for production?

Reply:
We don't use ActionMailer for any of the emails, could we just remove this?

Reply:
No, URL generators n AuthenticationMailer are using this...